docs/content/users/quickstart.md

```diff
@@ -222,10 +222,11 @@ DDEV comes ready to work with any PHP project, and has deeper support for severa
         ```bash
         mkdir my-typo3-site
         cd my-typo3-site
-        ddev config --project-type=typo3 --docroot=public --create-docroot
+        ddev config --project-type=typo3 --docroot=public --create-docroot --php-version 8.1
         ddev start
         ddev composer create "typo3/cms-base-distribution" --no-install
         ddev composer install
+        ddev config --auto
```

Comment:
Instead of `ddev config --auto` we also could recommend `ddev restart` to create the settings file. AFAIK this is a long standing issue anyway since the introduction of the `--no-install` option.

Reply:
Why would we do a `ddev config --auto` when we just did the explicit ddev config above? It doesn't seem like this belongs at all.

Reply:
Other solution would be to create the files even with a simple `ddev composer` command.

Reply:
I've checked the sources again, at the moment the settings files are create with start, config and importdb. Because I now need TYPO3 properly to be installed for the detection of the version, the behavior has changed a bit and doesn't currently work without an additional line her.

Reply:
Settings file is created on `ddev start` if the project type is set.

Reply:
Hope this should be clear now after your discussion yesterday and finally it's solved with https://github.com/drud/ddev/pull/4273.

Reply:
No, I never did understand, but glad about #4273.

Reply:
Oh, I see now. A `ddev restart` after the `ddev composer install` would have worked and is probably better, especially wrt mutagen. But we can go with this as it is now.